014-quarkus-panache-with-transactions-xa/src/test/java/io/quarkus/qe/rest/data/UserResourceTest.java

```diff
@@ -26,7 +26,7 @@ void testAllRepositoryMethods() {
         //Assert that order matches. UserRepository override it to be ascend. Initial order defined by import.sql
         String userList = given()
                 .accept("application/hal+json")
-                .when().get("/users/all")
+                .when().get("/users/all?sort=name")
```

Comment:
This can stay as is (i.e `when().get("/users/all)`)
The behavior of `get("/users/all?sort=!name")` can be checked, to verify it's sorting in descent order.